language/libra-tools/libra-events-fetcher/src/lib.rs

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::{bail, Result};
+use libra_json_rpc_client::{views::EventView, JsonRpcBatch, JsonRpcClient, JsonRpcResponse};
```

Comment:
Could you use libra_json_rpc_client::asych_client::Client instead, I'm going to deprecate old client